test/api/accounts.js

```diff
@@ -4,6 +4,10 @@ var node = require('./../node.js');
 
 var account = node.randomAccount();
 
+before(function (done) {
+	require('./../common/globalBefore').waitUntilBlockchainReady(done);
+});
+
```

Comment:
Running the blockchain loaded check in https://github.com/LiskHQ/lisk/blob/development/test/node.js will introduce the need of loading it asynchronously. The condition will have to be run every time when requiring the module. After all, I propose to leave the hooks as they are now.